### DISCUSSION TITLE
PyBaMM and MATLAB

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#3512] [Q&A]
Hello PyBaMM Team, 

Is there any mechanism that we can link PyBaMM results (EIS, Terminal voltage, SOC) to MATLAB or generate file that can be used in MATLAB. I am thinking of using PyBaMM model as benchmark model in MATLAB environment.

Any comments?

Thanks.

**Comment by brosaplanella** [comment]
PyBaMM outputs can be exported to a .mat file, see [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-6-managing-simulation-outputs.html) on how to do it.

**Comment by TomTranter** [comment]
We also have a simulink plugin which we'd be happy to discuss if you're interested. https://ion-works.com/products Still requires us making the model at this point or you sending us a pybamm script so don't have an automated service but we can send you an example file to play with

**Reply by Akila1993** [reply]
Hi @TomTranter Thank you very much for your reply. I am very much interested have discussion with you. I think your Simulink plugin will be ideal for my work. I will contact you with the pybamm script very soon. 
Also at some point we may have discussion too. 

Thank you once again.

**Reply by Akila1993** [reply]
@TomTranter I have sent the file via email, could you please check it.
